src/common/types/navigation/params/onboarding-stack-param.ts

```diff
@@ -1,9 +1,12 @@
 import { OnboardingRoute } from 'src/common/enums';
 import { ICareer } from '../../career';
+import { IEducation } from '../../education';
 
 type OnboardingStackParamList = {
   [OnboardingRoute.ONBOARDING]: undefined;
-  [OnboardingRoute.ADD_EDUCATION]: undefined;
+  [OnboardingRoute.ADD_EDUCATION]: {
+    _education?: IEducation;
```

Comment:
Why does education start with _ ?

Reply:
This was done because of the following error - I changed the name of the parameter to avoid the error.
![image](https://user-images.githubusercontent.com/41474876/161057056-7106a249-4f65-43a9-bd22-f8aa5f2b8fb0.png)

Reply:
Renamed in #80